test/no_repeats_please_test.js

```diff
@@ -0,0 +1,80 @@
+var testFile = require('../challenges/no-repeats.js');
+var mocha = require('mocha');
+var expect = require('expect.js');
+
+
+var runTest = function(spec) {
+    it(spec.description, function() {
+        expect(spec.input).to.equal(spec.result);
+    });
+};
+
+describe('repeats test suite', function() {
+    runTest({
+        description: 'repeats("aab") should return a number.',
+        input: typeof testFile.repeats('aab'),
```

Comment:
I think a better test design in general is to have the actual test function invocation done inside of the "it" block.  If the function happens to throw an exception (because it was passed a number instead of a string say), currently it halts the entire test run.  If the function invocation is done inside the it block, then any exceptions raised kill only that one test.  

this can be refactored in a later test